crates/burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -726,20 +726,53 @@ where
     }
 
     /// Returns the `k` largest elements of the given input tensor along a given dimension.
-    pub fn topk(self, k: usize, dim: usize) -> Tensor<B, D, K> {
+    pub fn topk(self, k: usize, dim: usize, largest: Option<usize>) -> Tensor<B, D, K> {
```

Comment:
Since we're only adding support for opset 1, the "largest" option is not necessary. We can remove the changes to the tensor API.

In any case, it would have been preferable to add it as another method, similar to `sort` & `sort_descending`. Something like `topk` and `topk_smallest`. But not required for this PR 🙂

Reply:
sounds good will remove the tensor api changes as well